src/other/precompile.jl

```diff
@@ -16,2236 +16,2236 @@ function precompile(all=false)
     if !all
         ccall(:jl_generating_output, Cint, ()) == 1 || return nothing
 
-        let fbody = try __lookup_kwbody__(which(DataFrames.combine, (GroupedDataFrame{DataFrame},Array{Bool,1},Vararg{Union{Regex, AbstractString, Function, Signed, Symbol, Unsigned, Pair, AbstractArray{T,1} where T, Type, All, Between, Cols, InvertedIndex},N} where N,))) catch missing end
+        let fbody = try __lookup_kwbody__(which(DataFrames.combine, (GroupedDataFrame{DataFrame}, Array{Bool, 1}, Vararg{Union{Regex, AbstractString, Function, Signed, Symbol, Unsigned, Pair, AbstractArray{T, 1} where T, Type, All, Between, Cols, InvertedIndex}, N} where N,))) catch missing end
```

Comment:
I would leave-out precompile.jl from fixing formatting as it is auto-generated.

Reply:
Ahh, :+1: